src/js/edu-benefits/edu-benefits-entry.jsx

```diff
@@ -12,7 +13,7 @@ import routes from './routes.jsx';
 
 require('../../sass/edu-benefits.scss');
 
-const store = createStore(reducer);
+const store = createStore(reducer, compose(applyMiddleware(thunk), __BUILDTYPE__ === 'development' && window.devToolsExtension ? window.devToolsExtension() : undefined));
```

Comment:
Awesome! I had been using the React devtools but not the Redux one, great addition.